Project-II-Calculator/calculator/src/components/ButtonComponents/ButtonContainer.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import NumberButton from './NumberButton';
+import ActionButton from './ActionButton';
+import './Button.css';
+
+const ButtonContainer = (props) => {
+   // console.log('ButtonContainer props in, ', props);
+
+  return (
+    <div className = "btn-container">
+      <ActionButton {...props} buttonStyle = 'wide-btn format'  value = {"clear"} />
```

Comment:
This isn't very DRY. If you separated your number buttons and operator buttons out into different components, you could use the map function to render your components.